app/retail/templates/leaderboard.html

```diff
@@ -17,117 +17,133 @@
       {% include 'shared/nav.html' %}
     </div>
     <div class="row no-gutter">
-      <div class="col body">
+      <div class="col body gitcoin-background">
         <div class="container-fluid" id="leaderboard">
-          <div class="offset-md-1 col-sm-12 col-md-10">
-            <div class="row mt-0 mb-2 pt-1 podium">
+
+          <!-- PODIUM -->
+
+          <div class="row offset-md-1 col-sm-12 col-md-10 mt-0 mb-2 pt-1 podium text-center justify-content-center" style="background-color: #0d023b">
               {% for podium_item in podium_items %}
-                <div class="col-4 winner">
+                <div class="col-2 winner mt-5">
                   {% if not is_linked_to_profile %}
-                    <img class="img-fluid"  src="{{podium_item.avatar_url}}" >
+                    <a href="{{podium_item.url}}" target="_blank">
+                      <img class="img-fluid"  src="{{podium_item.avatar_url}}" >
+                    </a>
                   {% endif %}
                   <div class="container winner-content">
                     <div class="row align-items-center">
                       {% if forloop.counter == 1 %}
-                        <div class="col-4 winner-position winner-position-first"></div>
-                      {% else %}
-                      {% if forloop.counter == 2 %}
-                        <div class="col-4 winner-position winner-position-second"></div>
-                      {% else %}
-                      {% if forloop.counter == 3 %}
-                        <div class="col-4 winner-position winner-position-third"></div>
-                      {% endif %}{% endif %}{% endif %}
-
-                      <div class="col-8 " >
-                        <h5 style="color:white">
-                          {% if not is_linked_to_profile %}
-                          <a href="{{podium_item.url}}" target="_blank">
-                            {{podium_item.at_ify_username}}
-                          </a>
-                          {% else %}
-                            {{podium_item.at_ify_username}}
-                          {% endif %}
-                        </h5>
-                        <h2 style="color:#86eec4" > ${{podium_item.amount|floatformat:0}} </h2>
-                      </div>
+                        <!--<div class="col-12 winner-position winner-position-first"></div>-->
+                      {% endif %}
```

Comment:
Are we using this ? If not we can remove it

Reply:
removed the entire div